src/main/java/JDBC/HeroDao.java

```diff
@@ -7,6 +7,8 @@ public interface HeroDao {
 
     List<Hero> findByName(String name);
 
+    List<Hero> findByID(Long id);
```

Comment:
в теорії findById має повертати унікальний елемент. 
```suggestion
    Hero findByID(Long id);
```